Vision/tests/Snippet/Annotation/SafeSearchTest.php

```diff
@@ -145,6 +145,15 @@ public function testIsViolent()
         $this->assertEquals(sprintf('Image contains %s content.', 'violent'), $res->output());
     }
 
+    public function testIsRacy()
```

Comment:
This test is failing. You need to add a `racy` key to `$this->ssData`, defined starting on line 35.

Reply:
Good find, thanks a lot.